http/client/src/main/java/org/eclipse/rdf4j/http/client/SharedHttpClientSessionManager.java

```diff
@@ -30,7 +30,15 @@
  * @author James Leigh
  */
 public class SharedHttpClientSessionManager implements HttpClientSessionManager, HttpClientDependent {
-
+	/**
+	 * FIXME: issue #1271, workaround for OpenJDK 8 bug.
+	 * ScheduledThreadPoolExecutor with 0 core threads may cause 100% CPU usage.
+	 * Using 1 core thread instead of 0 (default) fixes the problem but wastes some resources.
+	 * 
+	 * @see <a href="https://bugs.openjdk.java.net/browse/JDK-8129861">JDK-8129861</a>
+	 */
+	private static final int cores = (System.getProperty("org.rdf4j.client.executors.jdkbug") != null) ? 1 : 0;
```

Comment:
I'd prefer org.eclipse.rdf4j.* , keeping in sync with the package naming.

Reply:
ok, done